website/client-old/js/services/costumeServices.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+(function(){
+  angular
+    .module('habitrpg')
+    .factory('Costume', costumeFactory);
+
+  costumeFactory.$inject = [
+    'Content'
+  ];
+
+  function costumeFactory(Content) {
```

Comment:
I am using `noActivePet` / `noActiveMount` as a fallback in the case that a user has a value in `profile.options.current(Pet/Mount)` that doesn't exist (probably caused by "magic")